Makefile

```
@@ -272,6 +276,16 @@ misspell:
 	fi
 	misspell -w -i unknwon $(GO_SOURCES_OWN)
 
+
+.PHONY: manpage
+manpage:
+	@hash go-md2man > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
+		$(GO) get -u github.com/cpuguy83/go-md2man; \
```

```suggestion
		GO111MODULE=off $(GO) get -u github.com/cpuguy83/go-md2man; \
```

this prevents modification of `go.sum` and `go.mod` when running the target.